### ISSUE TITLE
setMenu in ubuntu

### ISSUE THREAD

**Issue by GitNomster** [SimulatedGREG/electron-vue#730]
Ubuntu 16.04
Electron 3.0.5
When i add setMenu:
```
function createWindow () {
  /**
   * Initial window options
   */
  mainWindow = new BrowserWindow({
    height: 563,
    useContentSize: true,
    width: 1000
  })
  mainWindow.setMenu(null) // here
  mainWindow.loadURL(winURL)

  mainWindow.on('closed', () => {
    mainWindow = null
  })
}
```
and try to start dev mode - it crashed. If i build and launch it - error appears:
`
Segmentation fault (core dumped)
`

**Comment by XanderLuciano** [comment]
I believe this would be an issue with electron and not this template.

**Comment by GitNomster** [comment]
When i tried it on clear electron - no error occurs

**Comment by VladimirMerk** [comment]
If you pass an empty menu object as an argument, it works.
`import { Menu } from 'electron'
this.mainWindow.setMenu(new Menu())`

**Comment by GitNomster** [comment]
On windows this creates empty menu line. I overcame the error like so:
```
...
} else if (process.platform === 'linux') {
  Menu.setApplicationMenu(Menu.buildFromTemplate([]))
} else {
  Menu.setApplicationMenu(null)
}
```

**Comment by GitNomster** [comment]
This was fixed in electorn 3.0.8.
